### DISCUSSION TITLE
How to show git deleted files?

### DISCUSSION THREAD

**Discussion by Diaoul** [nvim-tree/nvim-tree.lua#1173] [Q&A] [answered]
I'd like to show git files that are deleted but they don't show up in the tree as they are not on the disk anymore (deleted).
How is this feature supposed to work? Am I missing an option?

**Comment by kyazdani42** [comment, marked as answer]
The tree doesn't show deleted git files unfortunately. You can only show an icon next to a directory where a file was removed from the git tree.
